### PR TITLE
Create celo-vault-dapp

### PR DESCRIPTION
**Pull request by aglamadrid19**
Celo MakeItMobile hackathon

By the Helpi Team

How to build your first Vault Smart Contract and interface with it from a simple frontend.

### CODE REVIEW

**Comment by pyr0gan** [comment]
A description would be helpful. Is this an entry for the Celo Hackathon?

**Comment by aglamadrid19** [comment]
Description updated, updates are coming to this pull request , as we went to main net today. The pull request was just to make the submission the full tutorial should be finished today.

**Comment by aglamadrid19** [comment]
I have updated the tutorial with most of the content. We are only missing documenting the withdraw from the Vault

File: network-documentation/celo/tutorial/celo-vault-dapp.md [hunk 1]
```diff
@@ -0,0 +1,614 @@
...
+----                 -------------         ------ ----
+-a---           4/17/2021  7:35 AM         870803 Vault.json
+```
+
+Our next step is to copy and paste some code to the App React Component.
+```
+
+```
+
+After we are ready to start a development server test our APPROVE and LOCK button.
```
[the remark this answers is not in the document]
Comment (pyr0gan) [reply network-documentation/celo/tutorial/celo-vault-dapp.md:553]: @aglamadrid19 Can you please clarify this? Is code missing here?

File: network-documentation/celo/tutorial/celo-vault-dapp.md [hunk 2]
```diff
@@ -0,0 +1,614 @@
...
+    Directory: C:\Users\aglamadrid19\vault-dapp\my-vault-interface\src\contract
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+-a---           4/17/2021  7:35 AM         870803 Vault.json
+```
+
+Our next step is to copy and paste some code to the App React Component.
+```
+
```
[the remark this answers is not in the document]
Comment (aglamadrid19) [reply network-documentation/celo/tutorial/celo-vault-dapp.md:550]: Yes, I wasn't sure if I could update after the deadline. I'll complete the tutorial and get back to you . Thank you

File: network-documentation/celo/tutorial/celo-vault-dapp.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (aglamadrid19) [reply network-documentation/celo/tutorial/celo-vault-dapp.md:553]: Yes , I'll update fast . I wasn't sure I could update the tutorial after deadline.

[the remark this answers is not in the document]
Comment (yash-sharma1) [reply network-documentation/celo/tutorial/celo-vault-dapp.md:553]: @aglamadrid19 also can you please send your CELO wallet address to me or Rogan on Figment Discord

[the remark this answers is not in the document]
Comment (aglamadrid19) [reply network-documentation/celo/tutorial/celo-vault-dapp.md]: @pyr0gan @yash-sharma1 Quick update here, I added the react component code and docs for main functions. While in it we discovered a bug with the withdraw function of our vault smart contract. The bug is also affecting helpi prod so we plan on correcting it and update. For now the rest of the functionality works as intended (approve, lock, balance).

[the remark this answers is not in the document]
Comment (aglamadrid19) [reply network-documentation/celo/tutorial/celo-vault-dapp.md]: Thank you for the time, and patience. We should soon submit a curated / completed version.

**Comment by yash-sharma1** [comment]
@sytabaresa @aglamadrid19 looks good but can you please resolve the conflicting files so we can merge it 

Conflicting files
- SUMMARY.md
- network-documentation/celo/tutorial/README.md

@pyr0gan let's have a look if all good merge it once conflicting  files are resolved

**Comment by aglamadrid19** [comment]
We are ready for launch

**Pull request merged**